arbitrum-docs/node-running/how-tos/data-availability-committee/partials/_2.1-das-configuration-options-interfaces.mdx

```diff
@@ -0,0 +1,13 @@
+There are two main interfaces that can be enabled in a DAS: an **RPC interface** to store data in the DAS, intended to be used only by the AnyTrust sequencer; and a **REST interface** that supports only GET operations and is intended for public use.
+
+DA servers listen on the RPC interface for `das_store` RPC messages coming from the sequencer. The sequencer signs its requests and the DAS checks the signature. They also listen on the REST interface and respond to queries on `/get-by-hash/<hex encoded data hash>`. The response is always the same for a given hash.
+
+:::info
+
+Notice that it is possible to run one DAS that enables both interfaces, or one DAS with only the RPC interface enabled with an additional mirror DAS to serve REST requests.
```

Comment:
Same comment as before about the DAS needing to run the REST interface for the mirror to sync from, but maybe it's not exposed publicly. (Unless using the same S3 bucket backend for DAS and mirror, or using IPFS.)

Reply:
Modified the language to try to express this.